include/deal.II/grid/tria_levels.h

```diff
@@ -156,6 +156,16 @@ namespace internal
        */
       std::vector<types::subdomain_id> level_subdomain_ids;
 
+      /**
+       * Global cell index of each active cell.
+       */
+      std::vector<types::global_cell_index> global_active_cell_indices;
+
+      /**
+       * Global cell index of each cell on the given level.
+       */
+      std::vector<types::global_cell_index> global_level_cell_indices;
+
```

Comment:
Can you move these vectors close to the `active_cell_indices` field in line 103 or that part down here?